calico/networking/ip-autodetection.md

```diff
@@ -50,39 +50,92 @@ By default, {{site.prodname}} uses the **first-found** method; the first valid I
 - Address used by the node to reach a particular IP or domain (**can-reach**)
 - Regex to include matching interfaces (**interface**)
 - Regex to exclude matching interfaces (**skip-interface**)
+- A list of IP ranges in CIDR format to determine valid IP addresses on the node to choose from (**cidrs**)
 - Address assigned to kubernetes node (**kubernetes-internal-ip**)
 
+> **Note**: `kubernetes-internal-ip` is not available via the [Installation API]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.Installation).
```

Comment:
This is available now - we might just need to pull in the latest CRDs and API reference docs from the operator repo.

Reply:
Fixed. I'll update the API reference docs in a follow-up PR once I've gotten the `build-operator-reference` target sorted out.